Add setting to exit the application when Escape is pressed

mrubli opened the pull request:
This adds a configuration option to allow switching between the current Escape key behavior (minimizing the window) and a new behavior (exiting the application). I find this very helpful because it somewhat mimicks the _Everything_ behavior which can minimize to the tray area when Escape is pressed, at least until #52 is implemented.

Consider this a suggestion for now. If you're interested in merging this I'll double-check that the translation works and I'll add one or two translations as well.

I've also toyed with the idea of making the option a combobox like "Behavior after successfully opening a file" since the same three options would apply to the Escape key press behavior, so that would be a slightly different approach.

cboxdoerfer commented:
Thank you! Sorry, I'm a little busy at the moment. But I'll try to have a look at this PR this week.

mrubli commented:
Happy New Year! :-)

Have you had a chance to look at this PR yet? I noticed there are some merge conflicts but I'll be happy to resolve them if you're interested in this feature.

mrubli commented:
I finally got around to rebasing onto the latest master branch. :-) I've updated my feature branch, rebuilt and tested it, and updated the `NEWS` file as well.

cboxdoerfer commented:
Merged, thank you :)
